zipkin-server/src/main/resources/zipkin-server-shared.yml

```diff
@@ -209,6 +222,7 @@ server:
     min-response-size: 2048
 
 armeria:
+  healthCheckPath: /health
```

Comment:
I needed this to make the eureka integration in armeria use our composite endpoint we control. Otherwise, it wants to use /internal/health which won't include the full status. armeria could introduce a eureka health check path option, if it wanted, which would allow users to set this specific only to that part. Doing so would avoid a double-registration warning in the logs, which is distracting but has no other bad effect.